ui/app/routes/vault/cluster/license.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+import ClusterRoute from 'vault/mixins/cluster-route';
+
+export default Route.extend(ClusterRoute, {
+  model() {
+    return this.store.queryRecord('license', {});
```

Comment:
We need to decide how this should work in OSS and implement that here (or in the `beforeModel` hook) - do we show an upgrade message? Just redirect them back to somewhere else?